Paltineanu Rares-Mihai/3.9 - curs 4/main.c

```diff
@@ -0,0 +1,257 @@
+/**Write a program to perform the operations +,-,*, /  on two polynomials:
+A(x)=a0+a1x1+...+ anxn
+B(x)=b0+b1x1+......+ bmxm
+The degrees and the coefficients are read from the keyboard
+*/
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
```

Comment:
Really long class with way too much stuff going on, but I haven't taught you guys functions yet and modular programming so it's totally reasonable.

Great job!

Reply:
I just found out that i can respond to your comments. I would like to use functions, but i thought that it is better to code only in main until you will teach us functions next week. I will work with functions if it is ok.

Reply:
Sure it is